EIPS/eip-2982.md

```diff
@@ -0,0 +1,197 @@
+---
+eip: 2982
+title: Serenity Phase 0
+author: Danny Ryan (@djrtwo), Vitalik Buterin (@vbuterin)
+discussions-to: https://ethereum-magicians.org/t/serenity-phase-0-eip/4621
+status: Draft
+type: Standards Track
+category: Core
```

Comment:
```suggestion
type: Informational
```
We discussed a bit among some of the editors and it sounds like the weak consensus is that moving this to informational will unblock the primary concerns.  I'll try to give this a real review once that is done, though Informational EIPs I'm not sure have many hard requirements.

Reply:
I've made a number of suggestions below, most fairly small.  My general logic in asking for these changes is that this is an Informational EIP offered in the form of a technical proposal.   As such I want to keep it to technical information, not history or commitments to future action.  Especially, commitments as to what the Eth1 core developers will do in the future.

Reply:
thank you!